### ISSUE TITLE
Pre-trained weights trainings

### ISSUE THREAD

**Issue by ZJaume** [mozilla/firefox-translations-training#1416]
Model continuation for teachers is not working and I'm looking at adding support. But for now I'm experimenting with local trainings to see the effect of using `--pretrained-model` weights option with a similar language pair as starting point.

## Galician-English
Experiment using a pre-trained Spanish-English model
| training config | train steps | flores test chrF | flores test COMET
| ---- | ----- | --- | ---- |
| pipeline default | 380k | 66.9 | 88.01  |
| pretrain v1 | 6k | 66.5* | 87.51* |
| pretrain v2 | 42k | 67.0 | 87.92 |
| pretrain v3** | 45k | 66.8* | 87.75* |

\* means it is statistically significant compared to baseline (pipeline)

\*\* training is incomplete, has not finished or needs a fix

- v1: using pre-trained weights and do a normal training without chaning any parameters. No backtranslations, standalone marian without opustrainer.
- v2: same as v1 but train in a finetune-like manner, starting with a lower learning rate (3e-5 in this case). 
- v3: same as v2 but using opustrainer and backtranslations, trying to be as close as possible to pipeline setup. Keep low learning rate from v2.

## English-Galician
| training config | train steps | flores test chrF | flores test COMET
| ---- | ----- | --- | ---- |
| pipeline default | 160k | 62.4 | 86.35
| pretrain v3 | 141k | 63.4 | 87.25
| pretrain v3 new vocab |  116k | 62.1 | 86.16 |

## English-Basque
| training config | train steps | flores test chrF | flores test COMET
| ---- | ----- | --- | ---- |
| pipeline default | 168k | 60.0 | 86.98 |
| pretrain v3 | 90k | 59.8 | 86.95 |
| pretrain v3 new vocab | 82k | 58.8 | 86.05 |

Results are provisional, table will be updated as experiments progress.
